src/lib.rs

```diff
@@ -428,6 +428,22 @@ impl Build {
         self
     }
 
+    /// removes a compiler flag that was added by [`Build::flag`]
+    ///
+    /// # Example
+    /// ```no_run
+    /// cc::Build::new()
+    ///     .file("src/foo.c")
+    ///     .flag("unwanted_flag")
+    ///     .remove_flag("unwanted_flag")
+    ///     .compile("bar");
+    /// ```
```

Comment:
I think it's better to remove `.compile` so that the doc can be run.
While it cannot prevent the bug that happened, it can at least verify that calling it at runtime doesn't panic.

```suggestion
    /// ```
    /// cc::Build::new()
    ///     .file("src/foo.c")
    ///     .flag("unwanted_flag")
    ///     .remove_flag("unwanted_flag");
    /// ```
```